templates/telemetry_derived/clients_histogram_probe_counts_v1/query.sql

```diff
@@ -111,315 +111,344 @@ RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
 
-SELECT
-  CAST(NULL AS STRING) AS os,
-  CAST(NULL AS INT64) AS app_version,
-  CAST(NULL AS STRING) AS app_build_id,
-  channel,
-  metric,
-  metric_type,
-  key,
-  process,
-  agg_type AS client_agg_type,
-  'histogram' AS agg_type,
-  CAST(ROUND(SUM(record.value)) AS INT64) AS total_users,
-  udf_fill_buckets(udf_dedupe_map_sum(
-      ARRAY_AGG(record)
-  ), udf_to_string_arr(udf_get_buckets(first_bucket, last_bucket, num_buckets, metric_type))) AS aggregates
-FROM clients_histogram_bucket_counts_v1
-WHERE first_bucket IS NOT NULL
-GROUP BY
-  channel,
-  metric,
-  metric_type,
-  key,
-  process,
-  client_agg_type,
-  first_bucket,
-  last_bucket,
-  num_buckets
+  SELECT
+    CAST(NULL AS STRING) AS os,
+    CAST(NULL AS INT64) AS app_version,
+    CAST(NULL AS STRING) AS app_build_id,
+    channel,
+    metric,
+    metric_type,
+    key,
+    process,
+    agg_type AS client_agg_type,
+    'histogram' AS agg_type,
+    CAST(ROUND(SUM(record.value)) AS INT64) AS total_users,
+    udf_fill_buckets(udf_dedupe_map_sum(
+        ARRAY_AGG(record)
+    ), udf_to_string_arr(udf_get_buckets(first_bucket, last_bucket, num_buckets, metric_type))) AS aggregates
+  FROM clients_histogram_bucket_counts_v1
+  WHERE first_bucket IS NOT NULL
+  GROUP BY
+    channel,
+    metric,
+    metric_type,
+    key,
+    process,
+    client_agg_type,
+    first_bucket,
+    last_bucket,
+    num_buckets
+
+  UNION ALL
 
-UNION ALL
+  SELECT
+    CAST(NULL AS STRING) AS os,
+    CAST(NULL AS INT64) AS app_version,
+    CAST(NULL AS STRING) AS app_build_id,
+    CAST(NULL AS STRING) AS channel,
+    metric,
+    metric_type,
+    key,
+    process,
+    agg_type AS client_agg_type,
+    'histogram' AS agg_type,
+    CAST(ROUND(SUM(record.value)) AS INT64) AS total_users,
+    udf_fill_buckets(udf_dedupe_map_sum(
+        ARRAY_AGG(record)
+    ), udf_to_string_arr(udf_get_buckets(first_bucket, last_bucket, num_buckets, metric_type))) AS aggregates
+  FROM clients_histogram_bucket_counts_v1
+  WHERE first_bucket IS NOT NULL
+  GROUP BY
+    metric,
+    metric_type,
+    key,
+    process,
+    client_agg_type,
+    first_bucket,
+    last_bucket,
+    num_buckets)
 
 SELECT
-  CAST(NULL AS STRING) AS os,
-  CAST(NULL AS INT64) AS app_version,
-  CAST(NULL AS STRING) AS app_build_id,
-  CAST(NULL AS STRING) AS channel,
+  os,
+  app_version,
+  app_build_id,
+  channel,
   metric,
   metric_type,
   key,
   process,
-  agg_type AS client_agg_type,
-  'histogram' AS agg_type,
-  CAST(ROUND(SUM(record.value)) AS INT64) AS total_users,
-  udf_fill_buckets(udf_dedupe_map_sum(
-      ARRAY_AGG(record)
-  ), udf_to_string_arr(udf_get_buckets(first_bucket, last_bucket, num_buckets, metric_type))) AS aggregates
-FROM clients_histogram_bucket_counts_v1
-WHERE first_bucket IS NOT NULL
+  client_agg_type,
+  agg_type,
+  total_users,
+  ARRAY_AGG(STRUCT<bucket_type STRING, value ARRAY<STRUCT<key STRING, value FLOAT64>>>('default', aggregates)) AS aggregates
```

Comment:
So, this is a good point. If this change was for bucketing for histograms, then yes, I would be interested in the histogram type to decide how to bucket. But this change is just for scalars.

The only reason this file is getting changed is so that it also has a `bucket_type` added to it, but for now it's just `default`